.github/workflows/test.yml

```diff
@@ -13,11 +13,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ['2.7', '3.5', '3.6', '3.7', '3.8', '3.9', '3.10']
-        tarantool-version: [2.6, 2.7]
+        python-version: ['3.7', '3.8', '3.9', '3.10']
+        tarantool-version: ['2.8', '2.10']
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
     - name: update git submodules
```

Comment:
I think we can use the corresponding options in actions/checkout to do a submodules update.

Reply:
fixed.